org.lflang/src/org/lflang/federated/OldFedFileConfig.java

```diff
@@ -26,36 +22,39 @@
 package org.lflang.federated;
 
 import java.io.IOException;
+import java.nio.file.Path;
+
+import org.eclipse.emf.ecore.resource.Resource;
 
 import org.lflang.FileConfig;
 
 /**
  * A child class of @see FileConfig that extends the base functionality to add support
- * for federated execution. The code generator should create one instance of this class 
+ * for federated execution. The code generator should create one instance of this class
  * for each federate.
- * 
- * @author Soroush Bateni
  *
+ * @author Soroush Bateni
  */
-public class FedFileConfig extends FileConfig {
-    
+@Deprecated
```

Comment:
Shouldn't we just remove this instead of deprecating it? Why is it still needed?

Reply:
I have no idea. I'm all for removing it. Let me see if it's used anywhere. If not, I'll chunk it.

Reply:
Looks like it's still being used, but it's unclear to me why. It looks like nonsense to me. I'll just remove it and we'll see what breaks.